Add wagmi CLI

finessevanes opened the pull request:
Add the wagmi CLI to the Overview page so that devs can get started right away.

IljaDaderko commented on docs/web3modal/about.md:
```diff
@@ -1,3 +1,5 @@
+import ReactPlayer from 'react-player'
+
 # Web3Modal
```
Could you change this to "About" please. It shows as Web3Modal x2 times in the menu.

IljaDaderko commented on docs/web3modal/about.md:
```diff
@@ -8,6 +10,17 @@ For our supported flows, both react and HTML have great integration with [wagmi]
 
+## Wagmi CLI
+
+[wagmi](https://wagmi.sh/cli/create-wagmi) has created a CLI to get you started in just a few steps.
```
Too much about wagmi in Web3Modal about page imho. I would just leave 1 liner here, something like "Get started easily with wagmi-cli or see our installation docs for each platform". here `wagmi-cli` would just link to their page.

IljaDaderko replied:
Also removes the need of dependency on `react-player` (should be uninstalled)

chadyj reviewed (commented):
> Too much about wagmi in Web3Modal about page imho. I would just leave 1 liner here, something like "Get started easily with wagmi-cli or see our installation docs for each platform". here wagmi-cli would just link to their page.

Agree with Ilja here. A line should be enough.

Another alternative is a ["tip" admonition.](https://docusaurus.io/docs/next/markdown-features/admonitions)

finessevanes commented:
Okay, what do we think about the new layout.
1. Moved `About` page out for /React since this page will be entry point for every dev
2. I know you said less, wagmi, so I removed all the steps, but have it's own CLI section for readability
3. Added cards to give devs a quick way to examine which platform is right for them

IljaDaderko reviewed (changes requested):
1. I still think that dedicated `Getting started` section pointing to wagmi cli is out of scope for our documentation. Plus it is not relevant to non react platforms, so perhaps we can move it into "React" docs and present it as alternative at the end there?
2. This whole section is from older docs, so can be removed (all customisation options moved to Configuration page).

<img width="850" alt="Screenshot 2022-11-29 at 22 39 57" src="https://user-images.githubusercontent.com/3154053/204642890-29cc9e1e-ab4e-42bf-9250-07848dca74c0.png">

chadyj reviewed (changes requested):
<img width="351" alt="Screenshot 2022-11-30 at 10 46 12" src="https://user-images.githubusercontent.com/116099/204762829-fce4d234-6599-4119-b627-1b083a9cf9c6.png">

It is definitely better, but we shouldn't lead with an external tool.

Perhaps we could create a page called "Resources" or "Guides" like we have in the other sections, and add it there?
We could also add blog posts, or videos, or whatever content we have to help devs too.

And at a glance there isn't a big CTA to the demo? Maybe we can add a link to "try it out" in the resources or elsewhere?

@IljaDaderko Does that work for you?

finessevanes commented:
@chadyj My b, I didn't push my changes from my local, so that was old code you were looking at.

Lmk what you think about the latest updates.
